src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -62,10 +62,13 @@ static ucs_status_t uct_cuda_ipc_rkey_unpack(uct_component_t *component,
 
     UCT_CUDA_IPC_GET_DEVICE(cu_device);
 
-    status = UCT_CUDADRV_FUNC(cuDeviceCanAccessPeer(&peer_accessble,
-                                                    cu_device, packed->dev_num));
-    if ((status != UCS_OK) || (peer_accessble == 0)) {
-        return UCS_ERR_UNREACHABLE;
+    if (cu_device != packed->dev_num) {
```

Comment:
plz, remove it, it is changed the other way in #4137

Reply:
ah, thank you
done - after #4137 ucx_perftest's UCT tests will able work with cuda_ipc